src/antelope/chains/evm/telos-evm-testnet/index.ts

```diff
@@ -59,8 +61,14 @@ export default class TelosEVMTestnet extends EVMChainSettings {
         return { ...TOKEN, tokenId: this.constructTokenId(TOKEN) } as EvmToken;
     }
 
-    getUsdPrice(): Promise<number> {
-        return api.getCoingeckoUsdPrice('telos');
+    async getUsdPrice(): Promise<number> {
```

Comment:
yeah in the future we will refactor to be getFiat, but for now we only are able to get coingecko USD prices. so at the moment it would be inaccurate to call it getFiat. As for the reuse, the method needs to be implemented in each chain settings file because other chains in the future will use different logic